apps/qubit/modules/physicalobject/config/security.yml

```diff
@@ -1,13 +1,20 @@
+all:
+  is_secure: true
+
+autocomplete: 
+   credentials: [[ administrator, contributor, editor ]]
+
+boxList:
+   credentials: [[ administrator, editor ]]
+
 browse:
-  credentials: administrator
+   credentials: [[ administrator, editor, translator ]]
```

Comment:
Indent should be 2 spaces only.  Three spaces is used here and for every subsequent "credentials" line.